lib/src/base-styles/_nx-page-header.scss

```diff
@@ -76,14 +85,17 @@
   white-space: nowrap;
 }
 
-.nx-title__version {
+.nx-product__version {
+  @include bold();
+
   align-self: start;
   -ms-grid-row-align: start;
-  font-size: 12px;
+  font-size: 11px;
   -ms-grid-row: 3;
   -ms-grid-column: 3;
   grid-area: version;
   -ms-grid-column-align: left;
   justify-self: left;
-  line-height: 12px;
+  line-height: 11px;
+  padding-top: $nx-spacing-xxs;
```

Comment:
This seems like a hack to mitigate the height issues of the box above

Reply:
> This seems like a hack to mitigate the height issues of the box above

https://github.com/sonatype/sonatype-react-shared-components/pull/13/commits/ddbab71786239da9d3fc391fde51098a068df712